src/support/azext_support/commands.py

```diff
@@ -3,53 +3,54 @@
-        client_factory=cf_support_tickets)
-    support_communications = CliCommandType(
-        operations_tmpl='azext_support.vendored_sdks.operations#CommunicationsOperations.{}',
-        client_factory=cf_communications)
-
-    self.command_group('support', support, client_factory=cf_support)
-
-    with self.command_group('support services', support_services, client_factory=cf_services) as g:
-        g.command('list', method_name='list')
-        g.show_command('show', getter_name='get')
-
-    with self.command_group('support services problem-classifications', support_problem_classifications,
-                            client_factory=cf_problem_classifications) as g:
-        g.command('list', method_name='list')
-        g.show_command('show', getter_name='get')
-
-    with self.command_group('support tickets', support_tickets,
-                            client_factory=cf_support_tickets,
-                            deprecate_info=self.deprecate()) as g:
-        g.custom_command('list', 'list_support_tickets')
-        g.custom_show_command('show', 'get_support_tickets')
-        g.custom_command('create', 'create_support_tickets', supports_no_wait=False, validator=validate_tickets_create)
-        g.custom_command('update', 'update_support_tickets')
-
-    with self.command_group('support tickets communications', support_communications,
-                            client_factory=cf_communications,
-                            deprecate_info=self.deprecate()) as g:
-        g.custom_command('list', 'list_support_tickets_communications')
-        g.custom_show_command('show', 'get_support_tickets_communications')
-        g.custom_command('create', 'create_support_tickets_communications', supports_no_wait=False)
+    self.command_table["support in-subscription tickets update"] = TicketUpdate(
+        loader=self
+    )
+    self.command_table["support no-subscription tickets update"] = TicketUpdateNoSubscription(
+        loader=self
+    )
+    self.command_table["support in-subscription tickets create"] = TicketCreate(
+        loader=self
+    )
+    self.command_table["support no-subscription tickets create"] = TicketCreateNoSubscription(
+        loader=self
+    )
+    self.command_table["support in-subscription communication create"] = CommunicationCreate(
+        loader=self
+    )
+    self.command_table["support no-subscription communication create"] = CommunicationNoSubscriptionCreate(
+        loader=self
+    )
+    self.command_table["support in-subscription tickets list"] = TicketList(
+        loader=self
+    )
+    self.command_table["support no-subscription tickets list"] = TicketListNoSubscription(
+        loader=self
+    )
+    with self.command_group("support no-subscription file") as g:
+        g.custom_command("upload", "upload_files_no_subscription")
+
+    with self.command_group("support in-subscription file") as g:
+        g.custom_command("upload", "upload_files_in_subscription")
```

Comment:
Please add help in _help.py file for those two commands.

Reply:
Also provide help message for related parameters in _params.py

Reply:
Done